Formula/libwebm.rb

```diff
@@ -23,9 +23,12 @@ def install
       system "cmake", "..", *std_cmake_args
       system "make"
       lib.install "libwebm.a"
-      bin.install %w[sample sample_muxer vttdemux webm2pes]
+      bin.install %w[mkvparser_sample mkvmuxer_sample vttdemux webm2pes]
     end
     include.install Dir.glob("mkv*.hpp")
+    (include/"mkvmuxer").install Dir.glob("mkvmuxer/mkv*.h")
+    (include/"common").install Dir.glob("common/*.h")
+    (include/"mkvparser").install Dir.glob("mkvparser/mkv*.h")
     include.install Dir.glob("vtt*.h")
```

Comment:
Can we try to see if there's a `make install` now?

Reply:
There is no `make install`?

Reply:
Only for the new WebM Parser API https://github.com/webmproject/libwebm/commit/40da750807003000460e38f92fcb9d887834d715, which is disabled by default.
Installed files on local build with `-DENABLE_WEBM_PARSER:BOOL=ON`:
```
├── include
│   └── webm
│       ├── buffer_reader.h
│       ├── callback.h
│       ├── dom_types.h
│       ├── element.h
│       ├── file_reader.h
│       ├── id.h
│       ├── istream_reader.h
│       ├── reader.h
│       ├── status.h
│       └── webm_parser.h
└── lib
    └── libwebm.a
```